### PR TITLE
Create sequences_miray_cengil.py

### PR DESCRIPTION
**Pull request by miraycengil**
## Checklist
- [x] I have read the [CONTRIBUTING]
- [x] I have performed a self-review of my own code
- [x] I have run the code locally and it works as expected
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: Week03/sequences_miray_cengil.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    counts = {}
+    for item in seq:
+        if item in counts:
+            counts[item] += 1
+        else: 
+            counts[item] = 1
+    return counts
+
+def reverse_dict(d: dict) -> dict:
+    new_dict = defaultdict(list)
```
Comment (canbula) [inline Week03/sequences_miray_cengil.py:18]: You have an undefined name in this line.

**Review by canbula — CHANGES_REQUESTED** [review]
See the test results.

**Review by canbula — CHANGES_REQUESTED** [review]
I don't want you to import any modules.

**Comment by miraycengil** [comment]
Ok, sorry I used unnecessary modules in all my codes, I removed them all. I hope it's correct this time.

**Review by canbula — APPROVED** [review]
👍

**Pull request merged**